Data/Scripts/013_Items/004_1_PokeradarUI.rb

```diff
@@ -112,14 +112,20 @@ def createFusionIcon(pokemonId,x,y)
     bitmap1 = AnimatedBitmap.new(GameData::Species.icon_filename(headPoke))
     bitmap2 = AnimatedBitmap.new(GameData::Species.icon_filename(bodyPoke))
 
```

Comment:
```
    directory_name = "Graphics/Pokemon/FusionIcons"
    Dir.mkdir(directory_name) unless File.exists?(directory_name)
```
needs to be added, to check if directory exist, and create it if it doesn't exist. Game will crash without this code if the dir doesn't exist !